queries/lua/highlights.scm

```diff
@@ -121,6 +121,9 @@
 (function_call (identifier) @function . (arguments))
 (function_call (field_expression (property_identifier) @function) . (arguments))
 
+;; Parameters
+(function [(parameters (identifier))] @parameter)
```

Comment:
Why having only one alternative in here?

Reply:
why having `function` here? It's redundant.

Reply:
> Why having only one alternative in here? 

Wdym ?

Reply:
> why having `function` here? It's redundant. Also it does not apply to local_function

Done :+1: